colossalai/inference/server/completion_service.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+
+from colossalai.inference.core.async_engine import AsyncInferenceEngine
+
+from .utils import id_generator
+
+
+class ColossalAICompletionServing:
```

Comment:
no need to start with `ColossalAI`, just use `CompletitonServing` for simplicity.

Reply:
fixed